apps/exploits/serializers.py

```diff
@@ -21,6 +21,37 @@ class Meta:
         fields = ["cve", "product", "component", "affectedness", "resolution"]
 
 
+class TrackerReportDataSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tracker
+        fields = ["type", "external_system_id", "status", "resolution"]
+
+
+class AffectReportDataSerializer(serializers.ModelSerializer):
+    resolution = serializers.CharField(source="resolution_expanded")
+    trackers = TrackerReportDataSerializer(many=True, required=False)
+
+    class Meta:
+        model = Affect
+        fields = ["ps_module", "ps_component", "affectedness", "resolution", "trackers"]
+
+
+class FlawReportDataSerializer(serializers.ModelSerializer):
+    affects = AffectReportDataSerializer(many=True, required=False)
+
+    class Meta:
+        model = Flaw
+        fields = ["cve_id", "state", "resolution", "affects"]
+
+
+class ExploitReportDataSerializer(serializers.ModelSerializer):
+    flaw = FlawReportDataSerializer()
+
+    class Meta:
+        model = Exploit
+        fields = ["date", "source", "reference", "flaw"]
```

Comment:
**Tested for intended purpose:**
Please add missing fields "cve" (especially useful when flaw does not exist) and "maturity_preliminary" which will be needed for exploit sources different than CISA.